frontend/server/src/Controllers/Contest.php

```diff
@@ -1937,20 +1937,25 @@ private static function validateCommonCreateOrUpdate(
         $r->ensureOptionalBool('show_scoreboard_after');
 
         // languages is always optional
```

Comment:
Sabes que? Este comentario ya no coincide con lo que hace el código de abajo. Lo puedes quitar o indicar exactamente lo contrario
```suggestion
        // languages is always optional
```